dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -1252,6 +1252,7 @@ protected Node methodInvocationConvert(Node node, TypeMirror formalType) {
      *     to this method
      */
     protected List<Node> convertCallArguments(
+            Node expressionTypeReceiver,
```

Comment:
Add some Javadoc for the new parameter. From the name it isn't clear what Node this is.
It looks like null is allowed, so you should need to have `@Nullable` on the parameter.

Reply:
Thanks!